invenio_app_rdm/config.py

```diff
@@ -939,6 +939,8 @@ def github_link_render(record):
 RDM_COMMUNITIES_ROUTES = {
     "community-detail": "/communities/<pid_value>/records",
     "community-home": "/communities/<pid_value>/",
+    "community-static-page": "/communities/<pid_value>/info",
+    "community-join-waiting-list": "/communities/<pid_value>/join-waiting-list",
```

Comment:
minor: this should be handled by the `community-static-page` code above as invenio-app-rdm doesn't know about waiting lists

with @slint